sentry-android-core/src/main/java/io/sentry/android/core/SentryUserFeedbackWidget.java

```diff
@@ -0,0 +1,121 @@
+package io.sentry.android.core;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.os.Build;
+import android.util.AttributeSet;
+import android.util.TypedValue;
+import android.widget.Button;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class SentryUserFeedbackWidget extends Button {
```

Comment:
How about naming it Button, so people don't confuse it with the dialog?
```suggestion
public class SentryUserFeedbackButton extends Button {
```